docs/user/customization.md

```diff
@@ -0,0 +1,122 @@
+# Cluster Customization
+
+The OpenShift Installer allows for several different levels of customization. It is important to understand how and why each of these levels is exposed and the ramifications of making changes at each of these levels. This guide will attempt to walk through each of them and provide examples for why an administrator may want to make customizations.
+
+## Customization Levels
```

Comment:
The paragraph above is also talking about levels.  We can probably drop the header here (and bump levels for later headers appropriately) and just have two lead-in paragraphs in the same subsection.